lib/systems/default.nix

```
@@ -20,6 +21,15 @@ let
 
   inherit (lib.strings) toJSON;
 
+  # define this binding in the top-level let-block so that it only prints once
+  warnSystemEquals = lib.warn (lib.concatStringsSep " " [
```

I guess for consistency:

```suggestion
  warnPlatformEquals = lib.warn (lib.concatStringsSep " " [
```